docs/scripts/utils.py

```diff
@@ -86,6 +87,8 @@ def create_example_documentation(example_dest_dir: str, ignore_examples: bool):
         folder_name = sub_directory.stem
         formatted = " ".join(word.capitalize() for word in folder_name.split("_"))
 
+        # Copy the json file with the themes into the subfolder
+        shutil.copy(theme_json, sub_directory)
```

Comment:
why is this needed? cant the examples not just simply the file at its exact location?

Reply:
When converting a file to a jupyter notebook, the working directory of the jupyter notebook when being executed is exactly the location at which it is located. Thus, we cannot use the same path as in the plain example file.
In order to avoid exposing this to the users of the examples, I decided to solve this issue by not changing the path in the example file but handling this by just copying this file. For me, this felt like the easiest solution that does not cause any confusions for the users.

Reply:
@Scienfitz Does this make? If so, please resolve :)